telegram/ext/commandhandler.py

```diff
@@ -131,8 +140,17 @@ def __init__(self,
             if not re.match(r'^[\da-z_]{1,32}$', comm):
                 raise ValueError('Command is not a valid bot command')
 
-        self.filters = filters
-        self.allow_edited = allow_edited
+        if filters:
```

Comment:
Should we test that self.filters is a subclass of BaseFilter? Otherwise it will give an obscure crash when we try to & it.
Also, wouldn't this actually break simple function filters? Or have we not supported those for a while?

Reply:
we never checked before, we could introduce the check here but just let them get an error if they make a mistake imo.. function filters are removed in this version anyway